app/views/site/search_gems.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1>Results</h1>
+<ul>
```

Comment:
It would be great if this view should also include a search field in case the user didn't find the gem she was looking for or just want to search for another gem. What do you think?

You could put the search form in a partial and reuse it here and in `app/views/site/index.html.erb`.